sycl/include/sycl/ext/oneapi/matrix/matrix-jit.hpp

```diff
@@ -191,6 +191,21 @@ joint_matrix_mad(Group sg, joint_matrix<T1, M, K, LayoutA, Group> &mA,
                       PI_INVALID_DEVICE);
 #endif // __SYCL_DEVICE_ONLY__
 }
+
+template <typename Group, typename T, size_t NumRows, size_t NumCols,
+          matrix_layout Layout>
+inline __SYCL_ALWAYS_INLINE void
+joint_matrix_fill(Group sg,
+                  joint_matrix<T, NumRows, NumCols, Layout, Group> &res,
+                  const T v) {
```

Comment:
```suggestion
joint_matrix_fill(joint_matrix<T, NumRows, NumCols, Layout, Group> &res,
                          const T v) {
```

`sg` was unused

Reply:
Since we are not using scope on the SPIRV instruction as we are using the existing spirv_CompositeConstruct instruction. This argument will remain unused.
But we still need it on the DPC++ function to match the other DPC++ functions

Reply:
In this case please cast it to void, otherwise the compiler will emit a diagnostic.